dlf/plugins/pageview/tx_dlf_pageview.js

```diff
@@ -396,4 +444,215 @@ dlfViewer.prototype.initLayer = function(imageSourceObjs, isCorsEnabled) {
+        var rotation = viewerObject.map.getView().getRotation();
+
+        // fill form with coordinates
+        $('#addToBasketForm #startX').val(Math.round(pixel[0]));
+        $('#addToBasketForm #startY').val(Math.round(pixel[1]));
+        $('#addToBasketForm #endX').val(Math.round(pixel[2]));
+        $('#addToBasketForm #endY').val(Math.round(pixel[3]));
+        $('#addToBasketForm #rotation').val(Math.round(viewerObject.radianToDegree(rotation)));
+        //
+
+      	return geometry;
+    };
+
+    this.setNewCropDrawer(source);
+}
+
+/**
+ * reset crop selection
+ */
+dlfViewer.prototype.resetCropSelection = function () {
+	viewerObject = this;
+    this.map.removeLayer(this.selectionLayer);
+    this.source.clear();
+
+	var source = new ol.source.Vector({wrapX: false});
+
+	this.setNewCropDrawer(source);
+
+    this.selectionLayer = new ol.layer.Vector({
+    	source: source
+    });
+
+    this.map.addLayer(this.selectionLayer);
+    
+}
+
+dlfViewer.prototype.setNewCropDrawer = function  (source) {
+	viewerObject = this;
+	this.draw = new ol.interaction.Draw({
+      	source: source,
+      	type: /** @type {ol.geom.GeometryType} */ (value),
+      	geometryFunction: geometryFunction,
+      	maxPoints: maxPoints
+    });
+
+	// reset crop interaction
+    this.draw.on('drawend', function (event) {
+    	viewerObject.map.removeInteraction(viewerObject.draw);
+    });
+
+    this.selectionLayer = new ol.layer.Vector({
+    	source: source
+    });
+}
+
+/**
+ * add magnifier map
+ */
+dlfViewer.prototype.addMagnifier = function (rotation) {
+
+    //magnifier map
+    var extent = [0, 0, 1000, 1000];
+
+    layerProj = new ol.proj.Projection({
+        code: 'goobi-image',
+        units: 'pixels',
+        extent: extent
+    })
+
+    this.ov_view = new ol.View({
+        projection: layerProj,
+        center: ol.extent.getCenter(extent),
+        zoom: 3,
+        rotation: rotation,
+    });
+
+    this.ov_map = new ol.Map({
+      target: 'ov_map',
+      view: this.ov_view,
+      controls: [],
+      interactions: []
+    });
+    this.ov_map.addLayer(this.imageLayers[0]);
+    // var ov_layer = this.imageLayers;
+    // ov_map.addLayer(ov_layer);
+
+    var mousePosition = null;
+    var dlfViewer = this;
+
+    this.map.on('pointermove', function (evt) {
+        mousePosition = dlfViewer.map.getEventCoordinate(evt.originalEvent);
+        dlfViewer.ov_view.setCenter(mousePosition);
+    });
+
+    // $('#ov_map').hide();
+}
+
+/**
+ * activates the magnifier map
+ */
+dlfViewer.prototype.activateMagnifier = function () {
+    var rotation = this.map.getView().getRotation();
+    this.addMagnifier(rotation);
+    if (!this.magnifierEnabled) {
+        $('#ov_map').show();
+        this.magnifierEnabled = true;
+    } else {
+        $('#ov_map').hide();
+        // $('#ov_map').html('');
+        this.magnifierEnabled = false;
+    }
+}
+
+/**
+ * @constructor
+ * @extends {ol.interaction.Pointer}
```

Comment:
Take care that this also works with the openlayer custom build. Add `ol.interaction.Pointer` and all other *new* openlayers dependencies to [build config file](https://github.com/chrizzor/goobi-presentation/blob/107682ec9daa4810aa9d3969fdcb9ffda87a6984/dlf/lib/OpenLayers/ol3-dlf.json). The current build is based on the [slub fork of openlayers - branch: kitodo-presentation](https://github.com/slub/ol3/tree/kitodo-presentation).